examples/python/offline_vl.py

```diff
@@ -0,0 +1,34 @@
+import argparse
```

Comment:
We recommend presenting the usage in the user guide instead of adding an example, because the usage is the same among VLMs.